src/main/scala/Day1/Conditionals.scala

```diff
@@ -0,0 +1,19 @@
+package Day1
+
+object Conditionals extends App{
+
+  def conditionals(int1:Int, int2:Int, check:Boolean):Unit = {
+    if (check) {
+      val sum: Int = int1 + int2
+      println(sum)
```

Comment:
print the value directly `println(int1+int2)` no need to store it, same for the else block